index.html

```diff
@@ -16,18 +16,27 @@
           <img class="food-spin__logo" src="image/logofood.png" alt="logofood">
         </a>
         <nav>
-          <ul class="food-spin__menu">
-            <li>
-              <a href="#" class="food-spin__item js-food-spin-item" aria-label="see Breakfast">Breakfast</a>
-            </li>
-            <li>
-              <a href="#" class="food-spin__item js-food-spin-item" aria-label="see Lunch">Lunch</a>
-            </li>
-            <li>
-              <a href="#" class="food-spin__item js-food-spin-item" aria-label="see  Dinner">Dinner</a>
-            </li>
-          </ul>
-        </nav>
+          <nav>
+            <ul class="food-spin__menu">
+              <li class="food-spin__items">
+                <a href="#" class="food-spin__item js-food-spin-item" aria-label="see Breakfast">Breakfast</a>
+                <ul class="food-spin__submenu js-food-spin-submenu">
+                  <li><a href="#" class="food-spin__submenu-li" aria-label="see Eggs">Eggs</a></li>
+                  <li><a href="#" class="food-spin__submenu-li" aria-label="see Pancakes">Pancakes</a></li>
+                  <li><a href="#" class="food-spin__submenu-li" aria-label="see Omelette">Omelette</a></li>
+                  <li"><a href="#" class="food-spin__submenu-li" aria-label="see Breakfast Burrito">Breakfast Burrito</a></li>
+                </ul>
+              </li>
+              <li>
+                <a href="#" class="food-spin__item js-food-spin-item" aria-label="see Lunch">Lunch</a>
+              </li>
+              <li>
+                <a href="#" class="food-spin__item js-food-spin-item" aria-label="see Dinner">Dinner</a>
+              </li>
+            </ul>
+          </nav>
+          
```

Comment:
Al parecer escribiste mal aqui pues metiste el tag `nav` dentro del tag `nav` que ya tenias
```suggestion
          <ul class="food-spin__menu">
            <li class="food-spin__items">
              <a href="#" class="food-spin__item js-food-spin-item" aria-label="see Breakfast">Breakfast</a>
              <ul class="food-spin__submenu js-food-spin-submenu">
                <li><a href="#" class="food-spin__submenu-li" aria-label="see Eggs">Eggs</a></li>
                <li><a href="#" class="food-spin__submenu-li" aria-label="see Pancakes">Pancakes</a></li>
                <li><a href="#" class="food-spin__submenu-li" aria-label="see Omelette">Omelette</a></li>
                <li><a href="#" class="food-spin__submenu-li" aria-label="see Breakfast Burrito">Breakfast Burrito</a></li>
              </ul>
            </li>
            <li>
              <a href="#" class="food-spin__item js-food-spin-item" aria-label="see Lunch">Lunch</a>
            </li>
            <li>
              <a href="#" class="food-spin__item js-food-spin-item" aria-label="see Dinner">Dinner</a>
            </li>
          </ul>
```